mongodb-odbc-driver/installer/dmg/scripts/postinstall

```diff
@@ -1,82 +1,176 @@
-#!/usr/bin/python
-import errno
-import os
-import sys
+#!/usr/bin/env bash
 
-if sys.version_info.major == 2:
-    import ConfigParser as configparser
-else:
-    import configparser
+# Variables the installer application automatically passes along
+# $0 : Returns the path to the script
+# $1 : Returns the path to the package
+# $2 : Returns the target location (for example: /Applications)
+# $3 : Returns the target volume (for example: /Volumes/Macintosh HD)
 
-# Find the latest version directory. We do this because there
-# is no good way to pass the current version to the post install
-# script, and we don't want to have to update this script every
-# release.
-def get_latest(path):
-    vers = []
-    for d in os.listdir(path):
-        if not os.path.isdir(os.path.join(path, d)):
-            continue
-        ver_parts = d.split('.')
-        # All valid versions must contain 3 numeric parts.
-        if len(ver_parts) != 3:
-            continue
-        try:
-            ver_parts = map(int, ver_parts)
-        except:
-            continue
-        vers.append(ver_parts)
-    vers.sort()
-    return ".".join(map(str, vers[-1])) if len(vers) != 0 else ""
+LOGS_PATH="/tmp/postinstall_MongoDB_ODBC.log"
 
-ODBC_PATH       = '/Library/ODBC'
-INI             = '%s/odbcinst.ini'%ODBC_PATH
-INSTALL_ROOT    = '/Library/MongoDB/ODBC/'
-LATEST          = get_latest(INSTALL_ROOT)
-INSTALL_PATH    = INSTALL_ROOT + LATEST
-DRIVERS_SECTION = 'ODBC Drivers'
-MDB_UNICODE     = 'MongoDB Unicode ODBC %s'%LATEST
-MDB_ANSI        = 'MongoDB ANSI ODBC %s'%LATEST
-DRIVERS         = {
-                     MDB_UNICODE: '%s/libmdbodbcw.so'%INSTALL_PATH,
-                     MDB_ANSI:    '%s/libmdbodbca.so'%INSTALL_PATH
-                  }
+err(){
+    
+    echo "Err: $@" >> $LOGS_PATH
```

Comment:
```suggestion
    echo "Err: $@" | tee -a $LOGS_PATH
```
Would be helpful to also see the error in the console as well

Reply:
There actually is no console to log to.

Reply:
Right now, the installer does not relay logs from the postinstall script, but I'll add this just in case one day it does